packages/darma-vt/package.py

```diff
@@ -58,17 +58,73 @@ class DarmaVt(CMakePackage):
     version("1.0.0", tag="1.0.0")
     version("develop", branch="develop")
 
+    variant('lb_enabled', values=("0", "1"), default="1",
+            description='Compile with support for runtime load balancing')
+    variant('trace_enabled', values=("0", "1"), default="0",
+            description='Compile with support for runtime tracing')
+    variant('trace_only', values=("0", "1"), default="0",
+            description='Compile vt in trace-only mode')
+    variant('doxygen_enabled', values=("0", "1"), default="0",
+            description='Enable doxygen generation')
+    variant('mimalloc_enabled', values=("0", "1"), default="0",
+            description='Enable mimalloc, alternative allocator for debugging memory usage/frees/corruption')
+    variant('asan_enabled', values=("0", "1"), default="0",
+            description='Enable building with address sanitizer')
+    variant('werror_enabled', values=("0", "1"), default="0",
+            description='Treat all warnings as errors')
+    variant('pool_enabled', values=("0", "1"), default="1",
+            description='Use memory pool in vt for message allocation')
+    variant('zoltan_enabled', values=("0", "1"), default="0",
+            description='Build with Zoltan enabled for ZoltanLB support')
+    variant('mpi_guards', values=("0", "1"), default="0",
+            description='Guards against mis-use of MPI calls in code using vt')
+    variant('priorities_enabled', values=("0", "1"), default="1",
+            description='Enable prioritization of work')
+    variant('diagnostics_enabled', values=("0", "1"), default="1",
+            description='Enable VT component diagnostics for performance analysis')
+    variant('diagnostics_runtime_enabled', values=("0", "1"), default="0",
+            description='Enable VT component diagnostics at runtime by default')
+    variant('priority_bits_per_level', values=int, default=3,
+            description='Number of bits per level of priority in envelope')
+    variant('unity_build_enabled', values=("0", "1"), default="0",
+            description='Build with Unity/Jumbo mode enabled (requires CMake >= 3.16)')
+    variant('fcontext_enabled', values=("0", "1"), default="1",
```

Comment:
the default value in the docs is wrong for this (should be `0`)
I have a fix for this waiting (https://github.com/DARMA-tasking/vt/pull/1288/commits/681cdfc9fad11313e686e898d84d545a2ccc2837)

Reply:
Done.